app/components/funding_widget_component.html.erb

```diff
@@ -30,47 +30,33 @@
-            </div>
-          </div>
-          <div>
-            <h3>Next steps</h3>
-            <div>
-              <p>Use the <a href="https://www.gov.uk/student-finance-calculator">student finance calculator</a> to find out what tuition fee and maintenance loan you could get.<br></p>
-              <p>Explore <a href="/train-to-be-a-teacher">how to train to be a teacher</a>.</p>
+              <p>For this option, a scholarship or bursary is not available. You can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible.</p>
+              <p>You might be able to get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <%= render Content::InsetTextComponent.new(
+                header: "Non-UK citizens:",
+                text: "You're unlikely to get help funding your training unless you have permission to <a href=\"https://www.gov.uk/browse/visas-immigration/settle-in-the-uk\">live permanently in the UK</a>. Find out about <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">funding for non-UK citizens</a>.",
+                color: "purple-white"
+              ) %>
             </div>
           </div>
         <% else %>
           <div class="call-to-action__text">
             <h3><%= sub_head %></h3>
             <div>
               <%= helpers.safe_format(funding_results) %>
-
               <p>You can receive this alongside a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a>.</p>
-              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> and <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <%= render Content::InsetTextComponent.new(
+                header: "Non-UK citizens:",
+                text: "<p>If you train to teach languages or physics, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries and scholarships in other subjects.</p>
+                      <p>Other funding like tuition fee loans and maintenance loans are normally only available for UK citizens or non-UK citizens with permission to <a href=\"https://www.gov.uk/browse/visas-immigration/settle-in-the-uk\">live permanently in the UK</a>.</p>
+                      <p>Find out about <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">funding for non-UK citizens</a>.</p>",
+                color: "purple-white"
+              ) %>
             </div>
```

Comment:
Hey @martyn-w any idea why the paragraph breaks aren't picked up? I've displays without a space between them and I'm not sure why
![Screenshot 2024-08-09 at 15 44 37](https://github.com/user-attachments/assets/f9b7bc12-9ec7-4a72-810d-2bd48efd42da)

Reply:
Hi @jenhadfield2 I have investigated and the reason for the lack of spacing in these paragraphs is because the margin has been intentionally set to zero as part of this PR: https://github.com/DFE-Digital/get-into-teaching-app/pull/4058/commits/6b0df7ad86b44b3321559a4662f25edd90030acd  - I guess the aim here originally was to have a small section where the title runs close to the text?

The solution is either adjust the styles in `app/webpacker/styles/components/inset-text.scss` or add a new class with includes more margin.